addons/advanced_fatigue/functions/fnc_handleEffects.sqf

```diff
@@ -96,4 +96,4 @@ if (_overexhausted) then {
 _unit setVariable [QGVAR(aimFatigue), _fatigue];
 
 private _aimCoef = [missionNamespace, "ACE_setCustomAimCoef", "max"] call EFUNC(common,arithmeticGetResult);
-_unit setCustomAimCoef _aimCoef;
+[_unit, "ACE_setCustomAimCoef", _aimCoef, true] call EFUNC(common,setAimCoef);
```

Comment:
"ACE_setCustomAimCoef" is not a unique ID for AF. Would recommend `QUOTE(ADDON)` or something similar.

Reply:
pushed your suggestion